lib/neographics/src/text/text.c

```diff
@@ -98,20 +98,19 @@ void n_graphics_draw_text(
 
     uint32_t codepoint = 0, next_codepoint = 0, last_codepoint = 0,
         last_renderable_codepoint = 0, last_breakable_codepoint = 0;
+    
     while (text[index] != '\0') {
         // We're following the 2003 UTF-8 definition:
         // 0b0xxxxxxx
         // 0b110xxxxx 0b10xxxxxx
         // 0b1110xxxx 0b10xxxxxx 0b10xxxxxx
         // 0b11110xxx 0b10xxxxxx 0b10xxxxxx 0b10xxxxxx
-        if (text[index] == '\n'
-                && (char_origin.x + (__CODEPOINT_NEEDS_HYPHEN_AFTER(codepoint) ? hyphen->advance : 0)
-                        <= box.origin.x + box.size.w)) {
-            n_graphics_prv_draw_text_line(ctx, text,
-                line_begin, index, font, line_origin);
+        
+        if (text[index] == '\n' && (char_origin.x + (__CODEPOINT_NEEDS_HYPHEN_AFTER(codepoint) ? hyphen->advance : 0) <= box.origin.x + box.size.w)) {
```

Comment:
any particular reason why these lines were unbroken?

Reply:
No, I personally like it on one line and my editor was indenting me weird. I can revert